public_html/wp-content/themes/pattern-directory/header.php

```diff
@@ -27,10 +27,14 @@
 		<header id="masthead" class="site-header <?php echo is_home() ? 'home' : ''; ?>" role="banner">
 			<div class="site-branding">
 				<?php if ( is_home() ) : ?>
-					<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php echo esc_html_x( 'Patterns', 'Site title', 'wporg-patterns' ); ?></a></h1>
+					<div>
+						<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php echo esc_html_x( 'Patterns', 'Site title', 'wporg-patterns' ); ?></a></h1>
 
-					<p class="site-description"><?php esc_html_e( 'Add a beautifully designed, ready to go layout to any WordPress site with a simple copy/paste.', 'wporg-patterns' ); ?></p>
-					<?php get_search_form(); ?>
+						<p class="site-description"><?php esc_html_e( 'Add a beautifully designed, ready to go layout to any WordPress site with a simple copy/paste.', 'wporg-patterns' ); ?></p>
+						<?php get_search_form(); ?>
+						<a class="site-link" href="<?php echo esc_url( home_url( '/my-favorites' ) ); ?>"><?php esc_html_e( 'My favorites', 'wporg-patterns' ); ?></a>
+						<a class="site-link" href="<?php echo esc_url( home_url( '/my-patterns' ) ); ?>"><?php esc_html_e( 'My patterns', 'wporg-patterns' ); ?></a>
```

Comment:
I think we should omit the "My Patterns" link for now, since most people will see an empty state.

Reply:
Ah right, slipped my mind. 👍